src/channel-list-item.tsx

```diff
@@ -49,7 +49,17 @@ export class ChannelViewModel extends Model {
 
     when(this, x => x.model)
       .filter(c => isDM(c))
-      .switchMap(c => this.store.users.listen(c.user_id, c.api))
+      .switchMap(c => {
+        // XXX: This is a crime
+        let u = this.store.users.listen(c.user_id, c.api);
```

Comment:
This is the beginnings of the "I would like a model with These Fields Pleaseeeee" code, only poorly implemented